hr_attendance_modification_tracking/views/hr_attendance_view_tree.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="hr_attendance_view_tree" model="ir.ui.view">
```

Comment:
You should include this on `hr_attendance_view`,  no need of a new file

Reply:
Yes, and I would also add a filter in the search view, since at the end it is the purpose of this module

Reply:
OK, we have added to existing file.
Also added filter and group by.
Thanks.